src/lib.rs

```diff
@@ -52,37 +51,36 @@ impl<'a> MownStr<'a> {
         };
         MownStr {
             addr,
-            xlen: other.len(),
+            len: other.len(),
             _phd: PhantomData,
+            owners: None,
```

Comment:
This change goes against the aim of MownStr, which is to cost no more space than a regular `&str` or `Box<str>`. If we go for this cost, we might as well use `Cow<str>` instead of `MownStr`.